includes/wikia/services/ImageReviewService.class.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+use Swagger\Client\ImageReview\Api\ImageApi;
+use Wikia\DependencyInjection\Injector;
+use Wikia\Service\Swagger\ApiProvider;
+
+
+class ImageReviewService {
+	const SERVICE_NAME = 'image-review';
+
+	private $imageApiClient = null;
+
+	public function getImageHistory( $cityId, $pageId, $revisionId, User $user ):array {
+		$api = $this->getImageApiClient();
+		$api->getApiClient()
+			->getConfig()
+			->setApiKey('X-Wikia-UserId', $user->getId() ); //TODO: check if this is valid or should we use Access token
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)